pom.xml

```diff
@@ -64,6 +72,24 @@
         <version>1.3</version>
       </dependency>
       <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>junit</artifactId>
+        <version>1.10</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
(Note for me)
matrix-project depends on junit-plugin (via Jenkins core).
Jenkins core 1.580.1 bundles junit-1.0.